src/Stratis.Bitcoin.Networks/Stratis.Bitcoin.Networks.csproj

```diff
@@ -21,6 +21,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\NBitcoin\NBitcoin.csproj" />
+    <ProjectReference Include="..\Stratis.Bitcoin\Stratis.Bitcoin.csproj" />
```

Comment:
Is this needed? It shouldn't be.

Reply:
It is required to perform `ToUnixTimeStamp` in for the BIP9 parameter initialization. I didn't want the initialization to contain meaningless numbers as we are seeing elsewhere. `ToUnixTimeStamp` is found in `Stratis.Bitcoin.Utilities.Extensions`.